### PR TITLE
Smaller maintenance bundle: scope imports

### PR DESCRIPTION
**Pull request by rullzer**
Instead of include a lot of things we do not need. Scope the imports.
This way the maintenance bundle can be significantly smaller.

Before: 583Kb
Now: 39Kb

### CODE REVIEW

File: core/src/maintenance.js [hunk 1]
```diff
@@ -21,9 +21,9 @@
 
 import Axios from 'nextcloud-axios'
 
-import OC from './OC/index'
+import {getRootPath} from './OC/routing'
```
Comment (rullzer) [inline core/src/maintenance.js]: we could of course just directly import the webroot....

Comment (skjnldsv) [reply core/src/maintenance.js]: Yes!!! That's exactly what I was gonna do :D
You can even use the [OC alias](https://github.com/nextcloud/server/blob/89c701382f7a3dfaae3a45d1001ab26fb08cdd8a/webpack.common.js#L77) `import { Routing } from 'OC/routing'`

File: core/src/OC/routing.js [hunk 2]
```diff
@@ -174,4 +177,4 @@ export const filePath = (app, type, file) => {
  *
  * @since 8.2
  */
-export const getRootPath = () => OC.webroot
+export const getRootPath = () => webroot
```
Comment (ChristophWurst) [inline core/src/OC/routing.js]: IIRC I did this intentionally as our (legacy) jsunit tests do some weird mockery and thus rely on a overridden `OC.webroot` to pass.

In general this is bad code design as the tests do mangle with internals of a function call.

**Comment by ChristophWurst** [comment]
In general this is the correct approach. But our tests won't be happy.

**Comment by rullzer** [comment]
Lets do this in steps. But yeah this is the way to go.

**Pull request closed without merging**